KUBERNETES_CSI_OWNERS_ALIASES

```diff
@@ -31,15 +31,7 @@ aliases:
 
 # This documents who previously contributed to Kubernetes-CSI
 # as approver.
-emeritus_approver:
+emeritus_approvers:
 - lpabon
 - sbezverk
 - vladimirvivien
-
-# This documents who previously contributed to Kubernetes-CSI
-# as reviewer.
-emeritus_reviewer:
```

Comment:
The reviewers list should not be removed as it is not exactly the same as the approvers list.

Reply:
@xing-yang then you can track that using comments or something? `emeritus_reviewer` is not a supported yaml section in our tooling.

Reply:
Do we have emeritus_reviewers?

Reply:
@xing-yang nope. only thing allowed is `emeritus_approvers` / `approvers` / `reviewers`

Reply:
I see.